hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/DataXceiverServer.java

```diff
@@ -241,6 +241,13 @@ public void run() {
             .start();
       } catch (SocketTimeoutException ignored) {
         // wake up to see if should continue to run
+        if (peer != null) {
+          LOG.warn("A timeout occurred between DataXceiverServer: {} " +
```

Comment:
1. Let's rename the `ignored` to ste as now we are not ignoring it.
2. Let's not use warn level here. This seems not a concern for this timeout exception generally. I think info this level and `debug` for the `else` clause should be enough?